python-sdk/src/langwatch/prompts/__init__.py

```diff
@@ -0,0 +1,43 @@
+from .formatter import PromptFormatter, MissingPromptVariableError
+from .prompt import Prompt
+from typing import Any, Dict, Optional
+from opentelemetry import trace
+from ..langwatch_api_client import Client
+from ..langwatch_api_client.api.default import get_api_prompts_by_id
+from langwatch.telemetry.context import get_current_span
+from langwatch.attributes import AttributeKey
+from langwatch.state import get_instance
+
+
+tracer = trace.get_tracer(__name__)
+
+def get_prompt(prompt_id: str, version_id: Optional[str] = None) -> Dict[str, Any]:
+    """
+    Fetches a prompt config and formats it with the provided variables.
+    Returns a dict ready for OpenAI's client.
+    Raises MissingPromptVariableError if required variables are missing.
+    """
+    with tracer.start_as_current_span("get_prompt") as span:
+        span.set_attribute("inputs.prompt_id", prompt_id)
+        span.set_attribute("inputs.version_id", version_id) if version_id else None
+
+        try:
+            client = get_instance()
```

Comment:
`get_instance` can be `None`, which I think was actually a stupid decision on my part. Can you update the function of `get_instance` to call `ensure_setup`? Maybe refactor it to `ensure_instance`?

Reply:
ensure_setup calls get_instance...

```
def ensure_setup(api_key: Optional[str] = None) -> None:
    """Ensure LangWatch client is setup.

    If no client is setup, this will create a default client using environment variables.
    Validates that we have a working tracer provider to prevent silent failures.
    """
    client = get_instance()
```

Reply:
I think I got this. Please check @0xdeafcafe